components/StyledLink.js

```diff
@@ -106,6 +106,12 @@ StyledLink.propTypes = {
   truncateOverflow: PropTypes.bool,
   /** If true, the link will open in a new tab when clicked */
   openInNewTab: PropTypes.bool,
+  /** Color Shade, defaults to 500 */
+  colorShade: PropTypes.number,
+};
+
+StyledLink.defaultProps = {
+  colorShade: 500,
```

Comment:
I don't get why we need to customize this prop, links should normally always use the same color no?

Reply:
It was hardcoded to be 500 and 300 on hover.
I added a property to customize so I can update the colors if the background is a bit darker, like the collapsable transaction details where the background is grey.